### PR TITLE
initial commit for gallery restructure

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+
+| Notebook      | Author        | Description  |
+| ------------- |---------------| -------------|
+| AnomalyDetrend.ipynb | Will Wieder | TODO |
+| compare_esm_ssp_opt.ipynb | Katie Dagon | TODO |
+| LandUseChange_maps.ipynb | Dave Lawrennce | TODO |
+| MonthlySVD.ipynb | Will Wieder | TODO |
+| ScatterExample.ipynb | Danica Lombardozzi | TODO |
+| SimpleExample.ipynb | Will Wieder | TODO |
+| WeightedMeans.ipynb | Katie Dagon | TODO |
```
Comment (jhamman) [inline README.md]: @wwieder, @katiedagon, @djk2120, can you provide 1 line descriptions of each of these notebooks.

Comment (wwieder) [reply README.md]: @jhamman what's the easiest way to do this.  I'd like to just get the branch on my fork and directly modify the markdown file on github, or do I have to copy/ clone your form remotely?

Comment (wwieder) [reply README.md]: Never mind, edits can be made directly on Joe's branch, as i'd hoped...

**Comment by wwieder** [comment]
it looks like the cleaning of these notebooks removed the output from executed cells, but it seems like we'll want this in the gallery?

**Comment by wwieder** [comment]
How do we handle conda environments here?  I'm assuming that we all generated these notebooks using different environments?  Is this something that should be curated in the gallery?

File: README.md (hunk 1 above)
Comment (katiedagon) [reply README.md]: I edited the README.md to add my descriptions on @jhamman gallery branch and created a pull request. It doesn't show up here yet but I'm hoping I did it right.

**Comment by katiedagon** [comment]
> @jhamman what's an easy way I can visualize the diffs between our notebooks?

@wwieder I've used nbviewer to view jupyter notebooks using a github link: https://nbviewer.jupyter.org/

However, I can't figure out the right URL to give it to look at the diffs

**Comment by jhamman** [comment]
@wwieder - I plan to re-run all these notebooks after a bit more cleanup. We'll get the output back in there. 

To see the diff, go here: https://app.reviewnb.com/NCAR/ctsm_python_gallery/pull/26/

> How do we handle conda environments here? I'm assuming that we all generated these notebooks using different environments? Is this something that should be curated in the gallery?

Yes, I'm going to do my best to provide a comprehensive environment file for these notebooks.

> @wwieder I've used nbviewer to view jupyter notebooks using a github link: https://nbviewer.jupyter.org/

Yes! Perhaps we add a nbviewer link to the table of notebooks?

**Comment by wwieder** [comment]
<p>@jhamman, should we merge this in?  I'll have conflicts with several notebooks that I've continued to work on.  Finally, do I have permissions to be and administrator on this?</p><p> </p>

**Comment by wwieder** [comment]
These notebooks need to be rendered.  
Merging the PR will almost  certainly generate multiple conflicts with everyone's personal repositories from scripts they have been working on since @jhamman made the PR.  @billsacks should I go ahead and merge here, knowing we'll have lots of cleanup on Thursday?

**Comment by billsacks** [comment]
> @billsacks should I go ahead and merge here, knowing we'll have lots of cleanup on Thursday?

I have no objections, but I don't understand the full context here. Note that I don't think you'll be able to merge until you have fixed the conflicting files noted at the bottom of this PR. So you'll either need to do the merge on the command line and fix the conflicts there, or you'll need to merge the latest version of master into this branch, fix the conflicts, then you can merge back to master using the GitHub interface. (Unless GitHub now allows a way to merge a branch with conflicts and resolve the conflicts via the GitHub interface... I've never done that, but maybe it's possible.)

Writing this made me realize a potential issue for working with these python notebooks: If they have output embedded in them, they are ripe for merge conflicts that will probably be very difficult to resolve. Have you guys discussed a solution for this? A quick search turns up https://nextjournal.com/schmudde/how-to-version-control-jupyter, among others, but I haven't read through it carefully.

**Pull request closed without merging**